CRM/Contact/Page/View.php

```diff
@@ -306,7 +306,7 @@ public static function checkUserPermission($page, $contactID = NULL) {
    */
   public static function setTitle($contactId, $isDeleted = FALSE) {
     static $contactDetails;
-    $displayName = $contactImage = NULL;
```

Comment:
because it was that funny grey in phpstorm (ie. it's immediately re-defined)

Reply:
It would be grey because PHPStorm thinks it's being overwritten:

![image](https://user-images.githubusercontent.com/5212601/42929672-a7dd55aa-8b32-11e8-982c-315f381eb5d3.png)

But really it's just confused. I think we should either leave it in, or remove the $contactImage declaration too, by the same logic =]

Reply:
isn't the contactImage declaration removed?

Reply:
Nope, just on the next line down =]

Reply:
Yeah I agree with @JKingsnorth that we should remove both.